sympy/printing/ccode.py

```diff
@@ -219,8 +219,9 @@ def _print_ITE(self, expr):
         return self._print(_piecewise)
 
     def _print_MatrixElement(self, expr):
-        return "{0}[{1}]".format(expr.parent, expr.j +
-                expr.i*expr.parent.shape[1])
+        from sympy.printing.precedence import PRECEDENCE
```

Comment:
My suggestion to new contributors is always to use PyCharm as Python editor, it can handle imports automatically for example.